tasks.py

```diff
@@ -89,12 +89,21 @@ def imageid(ctx):
     )
 
 
-@task(name='create_db')
-def create_db(ctx):
-    """Call `create_all` on a SQLAlchemy database."""
+@task
+def revision(ctx, msg):
```

Comment:
Can we rename this to something more intuitive, like `add-migration`?  Running `invoke add-migration` to start a new migration from a template sounds more intuitive than `invoke revision`, especially if the developer isn't familiar with Alembic terminology.